src/account/account.ts

```diff
@@ -59,3 +81,30 @@ export async function createUser(
 
   return { ...account, reference }
 }
+
+/**
+ * Creates a new user (version 2) and uploads the encrypted account to the network
+ *
+ * @param connection connection information for data uploading
+ * @param username FDP username
+ * @param password FDP password
+ * @param mnemonic mnemonic phrase
+ */
+export async function createUser(
```

Comment:
Can you elaborate for me why do we need to store credentials in two steps?

Why can't the public key + password topic constuction for a SoC and the mnemonic encryption (with random data padding for the whole chunk) be enough?

Reply:
It looks like 2 steps to be able to change the password, but not be able to log into an account with two passwords. In the case of publicKey + password, it will be possible to log into the account with 2 passwords.